### ISSUE TITLE
v4: gap-x ScrollView contentContainerClassName 

### ISSUE THREAD

**Issue by tomasmozeris** [marklawlor/nativewind#671]
**Describe the bug**
in 4.0.13
```js 
<ScrollView contentContainerClassName="gap-16">
```
 used to add spacing between children and was very convenient as no need to worry about individual children spacing classNames. In 4.0.16 this functionality stopped working.

**To Reproduce**
```js 
<ScrollView contentContainerClassName="gap-16">
  <Text>1</Text>
  <Text>2</Text>
</Scrollview>
```

**Additional context**
Wondering if it is a some limitation of library from now on and I should start adding to each children className="my-8"

**Comment by johnf** [comment]
I think this might be fixed by #690

**Comment by marklawlor** [comment]
Fixed in `4.0.30`
